src/common/options/rgw.yaml.in

```diff
@@ -4271,3 +4271,12 @@ options:
   services:
   - rgw
   with_legacy: true
+- name: rgw_lua_enable
```

Comment:
please add:
```yaml
flags:
  - startup
```
to indicate the value need a restart

Reply:
done.